containers/podman-desktop-extension-ai-lab
# chore: use Table component from ui package

### What does this PR do?

This PR deletes the Table/SimpleColumn component and replace them with the one inside the ui package.

There was an issue though during the refactoring. The header of the table had a different color bc we customized it in ai lab.
There were two solutions to fix this:
1. update the Table in the ui package to allow customizing the color
2. update the background in ai-lab to be in sync with the rest of podman-desktop

I opted for the second as it will come in handy for the light mode

### Screenshot / video of UI

Nothing changes. I recorded this gif to show you the extension with the new bg color. It is the same used in desktop
![new_bg_color](https://github.com/containers/podman-desktop-extension-ai-lab/assets/49404737/c2578efb-c915-4920-96c5-e7e8ea8b066c)


### What issues does this PR fix or reference?

it resolves #969 

### How to test this PR?

1. nothing should change except some colors. Just navigate through ai lab to see if you notice some bad style
